hc/settings.py

```diff
@@ -135,47 +139,48 @@
 COMPRESS_OFFLINE = True
 
 EMAIL_BACKEND = "djmail.backends.default.EmailBackend"
+DJMAIL_REAL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
 
-# Slack integration -- override these in local_settings
+# Slack integration -- override these in dev_settings
 SLACK_CLIENT_ID = None
 SLACK_CLIENT_SECRET = None
 
-# Pushover integration -- override these in local_settings
+# Pushover integration -- override these in dev_settings
 PUSHOVER_API_TOKEN = None
 PUSHOVER_SUBSCRIPTION_URL = None
 PUSHOVER_EMERGENCY_RETRY_DELAY = 300
 PUSHOVER_EMERGENCY_EXPIRATION = 86400
 
-# Pushbullet integration -- override these in local_settings
+# Pushbullet integration -- override these in dev_settings
 PUSHBULLET_CLIENT_ID = None
 PUSHBULLET_CLIENT_SECRET = None
 
-if os.path.exists(os.path.join(BASE_DIR, "hc/local_settings.py")):
-    from .local_settings import *
+# Parse database configuration from $DATABASE_URL
+import dj_database_url
+
+DATABASES['default'] = dj_database_url.config()
+
+
+if os.path.exists(os.path.join(BASE_DIR, "hc/dev_settings.py")):
+    from .dev_settings import *
 else:
-    warnings.warn("local_settings.py not found, using defaults")
+    warnings.warn("dev_settings.py not found, using defaults")
 
 if os.environ.get('HEROKU'):
 
-    # Parse database configuration from $DATABASE_URL
-    import dj_database_url
-    DATABASES['default'] =  dj_database_url.config()
-
     # Honor the 'X-Forwarded-Proto' header for request.is_secure()
     SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 
     # Allow all host headers
     ALLOWED_HOSTS = ['*']
 
-    DJMAIL_REAL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
-    EMAIL_USE_TLS = bool(os.environ.get("EMAIL_USE_TLS"))
-    EMAIL_HOST = os.environ.get("EMAIL_HOST")
-    EMAIL_HOST_USER = os.environ.get("EMAIL_HOST_USER")
-    EMAIL_HOST_PASSWORD = os.environ.get("EMAIL_HOST_PASSWORD")
-    EMAIL_PORT = int(os.environ.get("EMAIL_PORT"))
+    try:
```

Comment:
That is what I have in heroku. Otherwise local settings for email vary across developers. Ideally everyone has their own settings in the file dev_settings.py, previously local_settings.py.